.dockerignore

```diff
@@ -1,4 +1,10 @@
 build/
 *.egg-info/
 venv/
-.git/
+# .git/ # Required for pyproject based automatic version detection
```

Comment:
Having the `.git` here was added due to a security finding (https://github.com/lyft/cartography/pull/1296).
Wondering if this can be changed to `.git/config` without breaking the build instead, as that file is the one with the issue.
A bit more context: 
* https://docs.github.com/en/actions/security-guides/automatic-token-authentication
* https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#stealing-the-jobs-github_token

Reply:
Hi @heryxpc thank you for the review.
Yes, that is a known issue and the risk is handled using multi stage docker build.

First stage `builder` gets all the contents but only uses them to build the python package which does not contain any of the Git files. 
The second stage `runner` only gets the build Python package and then install it. You can check the COPY command: https://github.com/lyft/cartography/pull/1300/files#diff-c66a06dd3419530dcc94979d2ffbd6afa551e1e0cdb4dacd3d9ccbea08d600bdR30

I went ahead and tried your suggestion of ignoring `.git/config`. The build passed locally.

Reply:
Github build has been successful too.